src/bootstrap/configure.py

```diff
@@ -161,12 +161,12 @@ def v(*args):
 # Many of these are saved below during the "writing configuration" step
 # (others are conditionally saved).
 o("manage-submodules", "build.submodules", "let the build manage the git submodules")
-o("full-bootstrap", "build.full-bootstrap", "build three compilers instead of two")
+o("full-bootstrap", "build.full-bootstrap", "build three compilers instead of two (not recommended except for testing reproducible builds)")
 o("extended", "build.extended", "build an extended rust tool set")
 
 v("tools", None, "List of extended tools will be installed")
 v("codegen-backends", None, "List of codegen backends to build")
-v("build", "build.build", "GNUs ./configure syntax LLVM build triple")
+v("build", "build.build", "GNUs ./configure syntax LLVM build triple (unsupported)")
```

Comment:
this should theoretically be supported, but it doesn't work in practice - I had a conversation a *long* time ago with someone trying to fix it, but can't find it. Not sure how to replicate locally, I don't have easy access to a musl machine.

Reply:
oh I forgot I still have access to @oli's machine - I'm not sure this is actually the right error though? if anyone knows how to test this happy to try it out
```
/home/joshua/rustc/build/x86_64-unknown-linux-musl/stage0/bin/cargo: error while loading shared libraries: /nix/store/9rabxvqbv0vgjmydiv59wkz768b5fmbc-glibc-2.30/lib/libc.so: invalid ELF header
```